dv-export/python/query-changes/changes_fetcher.py

```diff
@@ -0,0 +1,124 @@
+import copy
+import logging
+import math
+from datetime import datetime
+from typing import Any
+
+from visier.api import ModelApiClient, QueryApiClient
+from visier.connector import VisierSession
+
+from constants import *
+
+logger = logging.getLogger(__name__)
+
+
+class ChangesFetcher:
+    """Fetches changes for full time existence of subject."""
+
+    DEFAULT_QUERY_LIMIT = 1000
+
+    # Additional properties to fetch for different object types.
+    additional_properties = {
```

Comment:
Why are these separate? Could they not be baked into the query definition?

Reply:
We need a full list of properties with their types to be able to create the database table.